materials/sql-basic-queries.md

```diff
@@ -55,8 +55,24 @@ SELECT species_id, ROUND(hindfoot_length/1000.0, 2)
 FROM surveys;
 ```
 
-
 > Do [Exercise 2 - SELECT]({{ site.baseurl }}/exercises/Basic-queries-select-SQL).
+> Don't worry if you don't know how to save it yet, we'll cover that in a
+> minute.
+
+### Saving queries for future use
+
+* Views save queries to run again.
+* Create them by using `Create View` in the `View` menu, or by adding `CREATE
+  VIEW *viewname* AS` to the beginning of a query.
+
+```
+CREATE VIEW species_count AS
```

Comment:
Yes, totally. I hadn't originally written this up later and then realized they needed it for the assignment. Will fix shortly.